.github/workflows/dlp-pipelines.yml

```diff
@@ -165,21 +172,29 @@ jobs:
           DATASET_ID: ${{ needs.generate-uuid.outputs.output5 }}
         run: |
           gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV2 -Pargs=" \
+                --streaming --enableStreamingEngine \
                 --region=us-central1 \
                 --project=${{env.PROJECT_ID}} \
                 --tempLocation=gs://${{env.GCS_BUCKET}}/temp \
                 --numWorkers=2 \
                 --maxNumWorkers=3 \
                 --runner=DataflowRunner \
-                --filePattern=gs://${{env.GCS_BUCKET}}/${{env.INPUT_FILE_NAME}}.csv \
+                --filePattern=gs://${{env.GCS_BUCKET}}/${{env.INPUT_FILE_NAME}}*.csv \
                 --dataset=${{env.DATASET_ID}} \
                 --workerMachineType=n1-highmem-4 \
                 --inspectTemplateName=${{env.INSPECT_TEMPLATE_PATH}} \
                 --deidentifyTemplateName=${{env.DEID_TEMPLATE_PATH}} \
                 --batchSize=200000 \
                 --DLPMethod=DEID \
                 --serviceAccount=${SERVICE_ACCOUNT_EMAIL} \
-                --jobName=${{env.DEID_JOB_NAME}}"
+                --jobName=${{env.DEID_JOB_NAME}} \
+                --gcsNotificationTopic=${{env.GCS_NOTIFICATION_TOPIC}}"
+          sleep 30s
+          if gsutil stat gs://$BUCKET_NAME/$FILE_NAME; then
```

Comment:
Was this tested for case when FILE_NAME is not present?

Reply:
Also, where are you defining FILE_NAME?

Reply:
Thanks for pointing that out.